src/scss/components/_file-upload.scss

```diff
@@ -74,7 +74,7 @@ we don't have strict BEM naming here.
       @include bootstrap-span(width, 2);
     }
     .col-sm-10 {
-      @include bootstrap-span(width, 10);
+      max-width: 100%;
```

Comment:
I'm not sure if it's a good idea to change this: i think these CSS rules are here because the default formio uses bootstrap, so then the templates use these `col-sm-XX` to specify how many columns an element spans. 

I think if we don't want this styling, it might be better to remove the class from the HTML instead of changing its css (and if needed add a new class with other styling).